draft-bbs-signatures.md

```diff
@@ -534,9 +532,9 @@ Procedure:
 
 1. signature_result = octets_to_signature(signature)
 
-2. (i1, i2,..., iR) = RevealedIndexes
+2. [(H_i1, msg_i1), ..., (H_iR, msg_iR)] = [(H_i, msg_i), if revealed_i = 1]
```

Comment:
I think this syntax needs to be explained more, in general we could elect to do more of a filter or map style function?

Reply:
```suggestion
2. [(H_i1, msg_i1), ..., (H_iR, msg_iR)] = [(H_i, msg_i), if revealed_i = 1]
```
```suggestion
2. [(H_i1, msg_i1), ..., (H_iR, msg_iR)] = [(H_1, msg_1, revealed_1), ..., (H_L, msg_L, revealed_L)].filter(item => item.revealed_i == true)
```

Reply:
```suggestion
2. [(H_i1, msg_i1), ..., (H_iR, msg_iR)] = [(H_i, msg_i), if revealed_i = 1]
```
```suggestion
2. [(H_i1, msg_i1), ..., (H_iR, msg_iR)] = [(H_1, msg_1, revealed_1), ..., (H_L, msg_L, revealed_L)].filter(item => item.revealed_i == true)
```